crates/cheatcodes/src/credible.rs

```diff
@@ -223,12 +225,7 @@ pub fn execute_assertion(
         if let Some(expected) = &mut cheats.expected_revert {
             expected.max_depth = max(ecx.journaled_state.depth(), expected.max_depth);
         }
-        // Get a new inspector for logging
-        let mut inspector = executor.get_inspector(cheats);
-        inspector.console_log(&format!(
-            "Expected 1 assertion fn to be executed, but {total_assertions_ran} were executed."
-        ));
-        bail!("Assertion Fn number mismatch");
+        bail!("Expected 1 assertion to be executed, but {total_assertions_ran} were executed.");
```

Comment:
why are we doing this change?

Reply:
Basically a bunch of tests match against this exact error message and it's mentioned in the docs several times too. And it's unintuitive that the logs show one thing, but the revert reason is something else with the updated version. 
I think we can make the change eventually, but I don't think we should introduce breaking changes to the interface this shortly before a big release